setup.py

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env python3
 
 from setuptools import setup
+import sys
+
+# Requires python >= 3.3
+assert sys.version_info.major >= 3
+assert sys.version_info.minor >= 3
```

Comment:
@hobarrera 
> it still crashes on unsupported pythons. :(

I'd added code **to crash on unsupported pythons**.
Firstly as you know I'd modified README to use `pip3` but it's not standard command alias.
So within standard, I'd thought that guard is needed to block (crash) installation on unsupported python.
From my investigation, setuptools does not provide python version restriction, so assertion of python version may be better.